### PR TITLE
64 vector other crs

### PR DESCRIPTION
**Pull request by index-git**
Part of issue #64

- [x] Tests
- ~~Layman Test Client~~
- [x] Changlelog
- [x] Documentation

Optionally
- Add dependency to doc/dependencies.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ make timgen-build
 - [#64](https://github.com/LayerManager/layman/issues/64) Responses of [GET Layers](doc/rest.md#get-layers), [GET Workspace Layers](doc/rest.md#get-workspace-layers), [GET Workspace Layer](doc/rest.md#get-workspace-layer), [PATCH Workspace Layer](doc/rest.md#patch-workspace-layer), [GET Maps](doc/rest.md#get-maps), [GET Workspace Maps](doc/rest.md#get-workspace-maps), [GET Workspace Map](doc/rest.md#get-workspace-map), [PATCH Workspace Map](doc/rest.md#patch-workspace-map) contains new attribute
    - `native_crs` with code of native CRS in form "EPSG:&lt;code&gt;", e.g. "EPSG:4326"
    - `native_bounding_box` with coordinates in native CRS  [minx, miny, maxx, maxy]
-- [#64](https://github.com/LayerManager/layman/issues/64) Layman supports import of vector layers in "EPSG:5514" and "EPSG:32633".
+- [#64](https://github.com/LayerManager/layman/issues/64) Layman supports import of vector layers in "EPSG:5514", "EPSG:32633" and "EPSG:32633".
```
Comment (jirik) [inline CHANGELOG.md]: Rename one  `32633` to `32634`

File: doc/rest.md [hunk 2]
```diff
@@ -148,7 +148,7 @@ Body parameters:
    - by default it is layer name
 - *description*
    - by default it is empty string
-- *crs*, string `EPSG:3857`, `EPSG:4326`, `EPSG:5514` or `EPSG:32633`
+- *crs*, string `EPSG:3857`, `EPSG:4326`, `EPSG:5514`, `EPSG:32633` or `EPSG:32633`
```
Comment (jirik) [inline doc/rest.md]: Rename one  `32633` to `32634`

File: doc/rest.md [hunk 3]
```diff
@@ -299,7 +299,7 @@ Body parameters:
    - if QML style is used (either directly within this request, or indirectly from previous state on server), it must list all attributes contained in given data file
 - *title*
 - *description*
-- *crs*, string `EPSG:3857`, `EPSG:4326`, `EPSG:5514` or `EPSG:32633`
+- *crs*, string `EPSG:3857`, `EPSG:4326`, `EPSG:5514`, `EPSG:32633` or `EPSG:32633`
```
Comment (jirik) [inline doc/rest.md]: Rename one  `32633` to `32634`

File: src/crs/__init__.py [hunk 4]
```diff
@@ -148,4 +149,26 @@
...
+      <proj4>+proj=lcc +lat_0=52 +lon_0=10 +lat_1=35 +lat_2=65 +x_0=4000000 +y_0=2800000 +ellps=GRS80 +units=m +no_defs</proj4>
+      <srsid>999</srsid>
+      <srid>3034</srid>
```
Comment (jirik) [inline src/crs/__init__.py:165]: It should be 900915

File: src/crs/__init__.py [hunk 5]
```diff
@@ -148,4 +149,26 @@
...
+      <authid>EPSG:3034</authid>
+      <description>ETRS89-extended / LCC Europe</description>
+      <projectionacronym>lcc</projectionacronym>
+      <ellipsoidacronym>EPSG:7019</ellipsoidacronym>
+      <geographicflag>false</geographicflag>''',
+        axes_order_east_north_in_epsg_db=False,
+        proj4text='+proj=lcc +lat_1=35 +lat_2=65 +lat_0=52 +lon_0=10 +x_0=4000000 +y_0=2800000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs ',
+        srid=90015,
```
Comment (jirik) [inline src/crs/__init__.py:173]: It should be 900915

File: src/crs/__init__.py [hunk 6]
```diff
@@ -148,4 +149,26 @@
...
+      <proj4>+proj=lcc +lat_0=52 +lon_0=10 +lat_1=35 +lat_2=65 +x_0=4000000 +y_0=2800000 +ellps=GRS80 +units=m +no_defs</proj4>
```
Comment (jirik) [inline src/crs/__init__.py:163]: Should be same as proj4text below

File: src/crs/__init__.py [hunk 7]
```diff
@@ -171,4 +172,26 @@
...
+      <srsid>1000</srsid>
+      <srid>3035</srid>
+      <authid>EPSG:3035</authid>
+      <description>ETRS89-extended / LAEA Europe</description>
+      <projectionacronym>laea</projectionacronym>
+      <ellipsoidacronym>EPSG:7019</ellipsoidacronym>
+      <geographicflag>false</geographicflag>''',
+        axes_order_east_north_in_epsg_db=False,
+        proj4text='+proj=laea +lat_0=52 +lon_0=10 +x_0=4321000 +y_0=3210000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs ',
+        srid=90016,
```
Comment (jirik) [inline src/crs/__init__.py:195]: It should be 900916

File: src/crs/__init__.py [hunk 8]
```diff
@@ -171,4 +172,26 @@
...
+        world_bounds=dict(),
+        qgis_template_spatialrefsys='''<wkt>PROJCRS["ETRS89-extended / LAEA Europe",BASEGEOGCRS["ETRS89",DATUM["European Terrestrial Reference System 1989",ELLIPSOID["GRS 1980",6378137,298.257222101,LENGTHUNIT["metre",1]]],PRIMEM["Greenwich",0,ANGLEUNIT["degree",0.0174532925199433]],ID["EPSG",4258]],CONVERSION["Europe Equal Area 2001",METHOD["Lambert Azimuthal Equal Area",ID["EPSG",9820]],PARAMETER["Latitude of natural origin",52,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8801]],PARAMETER["Longitude of natural origin",10,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8802]],PARAMETER["False easting",4321000,LENGTHUNIT["metre",1],ID["EPSG",8806]],PARAMETER["False northing",3210000,LENGTHUNIT["metre",1],ID["EPSG",8807]]],CS[Cartesian,2],AXIS["northing (Y)",north,ORDER[1],LENGTHUNIT["metre",1]],AXIS["easting (X)",east,ORDER[2],LENGTHUNIT["metre",1]],USAGE[SCOPE["unknown"],AREA["Europe - LCC &amp; LAEA"],BBOX[24.6,-35.58,84.17,44.83]],ID["EPSG",3035]]</wkt>
+      <proj4>+proj=laea +lat_0=52 +lon_0=10 +x_0=4321000 +y_0=3210000 +ellps=GRS80 +units=m +no_defs</proj4>
+      <srsid>1000</srsid>
+      <srid>3035</srid>
```
Comment (jirik) [inline src/crs/__init__.py:187]: It should be 900916

File: src/crs/__init__.py [hunk 9]
```diff
@@ -171,4 +172,26 @@
...
+            6829874.45,
+        ),
+        no_area_bbox_padding=1,
+        world_bounds=dict(),
+        qgis_template_spatialrefsys='''<wkt>PROJCRS["ETRS89-extended / LAEA Europe",BASEGEOGCRS["ETRS89",DATUM["European Terrestrial Reference System 1989",ELLIPSOID["GRS 1980",6378137,298.257222101,LENGTHUNIT["metre",1]]],PRIMEM["Greenwich",0,ANGLEUNIT["degree",0.0174532925199433]],ID["EPSG",4258]],CONVERSION["Europe Equal Area 2001",METHOD["Lambert Azimuthal Equal Area",ID["EPSG",9820]],PARAMETER["Latitude of natural origin",52,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8801]],PARAMETER["Longitude of natural origin",10,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8802]],PARAMETER["False easting",4321000,LENGTHUNIT["metre",1],ID["EPSG",8806]],PARAMETER["False northing",3210000,LENGTHUNIT["metre",1],ID["EPSG",8807]]],CS[Cartesian,2],AXIS["northing (Y)",north,ORDER[1],LENGTHUNIT["metre",1]],AXIS["easting (X)",east,ORDER[2],LENGTHUNIT["metre",1]],USAGE[SCOPE["unknown"],AREA["Europe - LCC &amp; LAEA"],BBOX[24.6,-35.58,84.17,44.83]],ID["EPSG",3035]]</wkt>
+      <proj4>+proj=laea +lat_0=52 +lon_0=10 +x_0=4321000 +y_0=3210000 +ellps=GRS80 +units=m +no_defs</proj4>
```
Comment (jirik) [inline src/crs/__init__.py:185]: It should be same as proj4string below

File: src/crs/__init__.py [hunk 10]
```diff
@@ -148,4 +149,26 @@
...
...        qgis_template_spatialrefsys='''<wkt>PROJCRS["ETRS89-extended / LCC Europe",BASEGEOGCRS["ETRS89",DATUM["European Terrestrial Reference System 1989",ELLIPSOID["GRS 1980",6378137,298.257222101,LENGTHUNIT["metre",1]]],PRIMEM["Greenwich",0,ANGLEUNIT["degree",0.0174532925199433]],ID["EPSG",4258]],CONVERSION["Europe Conformal 2001",METHOD["Lambert Conic Conformal (2SP)",ID["EPSG",9802]],PARAMETER["Latitude of false origin",52,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8821]],PARAMETER["Longitude of false origin",10,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8822]],PARAMETER["Latitude of 1st standard parallel",35,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8823]],PARAMETER["Latitude of 2nd standard parallel",65,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8824]],PARAMETER["Easting at false origin",4000000,LENGTHUNIT["metre",1],ID["EPSG",8826]],PARAMETER["Northing at false origin",2800000,LENGTHUNIT["metre",1],ID["EPSG",8827]]],CS[Cartesian,2],AXIS["northing (N)",north,ORDER[1],LENGTHUNIT["metre",1]],AXIS["easting (E)",east,ORDER[2],LENGTHUNIT["metre",1]],USAGE[SCOPE["unknown"],AREA["Europe - LCC &amp; LAEA"],BBOX[24.6,-35.58,84.17,44.83]],ID["EPSG",3034]]</wkt>
```
Comment (jirik) [inline src/crs/__init__.py:162]: Try to update this WKT from epsg.io

File: src/crs/__init__.py [hunk 11]
```diff
@@ -171,4 +172,26 @@
...
+        world_bbox=(
+            1584884.54,
+            1507846.05,
+            8442721.99,
+            6829874.45,
+        ),
+        no_area_bbox_padding=1,
+        world_bounds=dict(),
+        qgis_template_spatialrefsys='''<wkt>PROJCRS["ETRS89-extended / LAEA Europe",BASEGEOGCRS["ETRS89",DATUM["European Terrestrial Reference System 1989",ELLIPSOID["GRS 1980",6378137,298.257222101,LENGTHUNIT["metre",1]]],PRIMEM["Greenwich",0,ANGLEUNIT["degree",0.0174532925199433]],ID["EPSG",4258]],CONVERSION["Europe Equal Area 2001",METHOD["Lambert Azimuthal Equal Area",ID["EPSG",9820]],PARAMETER["Latitude of natural origin",52,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8801]],PARAMETER["Longitude of natural origin",10,ANGLEUNIT["degree",0.0174532925199433],ID["EPSG",8802]],PARAMETER["False easting",4321000,LENGTHUNIT["metre",1],ID["EPSG",8806]],PARAMETER["False northing",3210000,LENGTHUNIT["metre",1],ID["EPSG",8807]]],CS[Cartesian,2],AXIS["northing (Y)",north,ORDER[1],LENGTHUNIT["metre",1]],AXIS["easting (X)",east,ORDER[2],LENGTHUNIT["metre",1]],USAGE[SCOPE["unknown"],AREA["Europe - LCC &amp; LAEA"],BBOX[24.6,-35.58,84.17,44.83]],ID["EPSG",3035]]</wkt>
```
Comment (jirik) [inline src/crs/__init__.py:184]: Try to update this WKT from epsg.io

**Review by jirik — CHANGES_REQUESTED** [review]

File: src/crs/__init__.py (hunk 6 above)
Comment (jirik) [reply src/crs/__init__.py:163]: I think we should add 3034 to epsg_properties.py

File: src/crs/__init__.py (hunk 9 above)
Comment (jirik) [reply src/crs/__init__.py:185]: I think we should add 3035 to epsg_properties.py

**Review by jirik — CHANGES_REQUESTED** [review]

**Comment by index-git** [comment]
Replaced by #571 571

**Pull request closed without merging**